base/rhel9-python-3.9/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM registry.redhat.io/rhel9/python-39:latest
+
+LABEL name="odh-notebook-base-rhel9-python-3.9" \
+      summary="Python 3.9 Red Hat Enterprise Linux 9 base image for ODH notebooks" \
+      description="Base Python 3.9 builder image based on Red Hat Enterprise Linux 9 for ODH notebooks" \
+      io.k9s.display-name="Python 3.9 RHEL9 base image for ODH notebooks" \
+      io.k9s.description="Base Python 3.9 builder image based on RHEL9 for ODH notebooks" \
+      authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
+      io.openshift.build.commit.ref="main" \
+      io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/base/rhel9-python-3.9" \
+      io.openshift.build.image="quay.io/opendatahub/workbench-images:base-rhel9-python-3.9"
```

As this is exclusive to the downstream repository, best to have this configured exclusively for this repo.

```suggestion
LABEL name="rhoai-notebook-base-rhel9-python-3.9" \
      summary="Python 3.9 Red Hat Enterprise Linux 9 base image for RHOAI notebooks" \
      description="Base Python 3.9 builder image based on Red Hat Enterprise Linux 9 for RHOAI notebooks" \
      io.k9s.display-name="Python 3.9 RHEL9 base image for RHOAI notebooks" \
      io.k9s.description="Base Python 3.9 builder image based on RHEL9 for RHOAI notebooks" \
      authoritative-source-url="https://github.com/red-hat-data-services/notebooks" \
      io.openshift.build.commit.ref="main" \
      io.openshift.build.source-location="https://github.com/red-hat-data-services/notebooks/tree/main/base/rhel9-python-3.9" \
      io.openshift.build.image="quay.io/modh/odh-base-rhel9"
```

---

Maybe we should introduce this file structure in upstream.
Not build it, just the file structure.
so that we can follow the same pattern.
WDYT?

---

I think it would be a good practice.